debian/generate_changelog.sh

```
@@ -0,0 +1,7 @@
+#!/usr/bin/bash
+
+export DEBEMAIL="info@specterops.io"
+export PKG_VERSION=$(git describe --tags --match "v*" | sed 's/^v//g')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)